app/services/update_auction.rb

```diff
@@ -10,11 +10,31 @@ def perform
   end
 
   def should_create_cap_proposal?
-    attributes[:result] == 'accepted' && auction.cap_proposal_url.blank?
+    auction_accepted_and_cap_proposal_is_blank? &&
+      winning_bidder_is_eligible_to_be_paid?
   end
 
   private
 
+  def auction_accepted_and_cap_proposal_is_blank?
+    attributes[:result] == 'accepted' && auction.cap_proposal_url.blank?
+  end
+
+  def winning_bidder_is_eligible_to_be_paid?
+    auction_presenter = AuctionPresenter.new(auction)
```

Comment:
I'm fine with that, but I need access to `winning_bid`/winning bidder. What would be a good way to get that without `AuctionPresenter`?